src/components/CourseChecklist/index.jsx

```diff
@@ -327,6 +332,13 @@ class CourseChecklist extends React.Component {
     return false;
   }
 
+  shouldShowCheck = (checkID) => {
+    if (checkID === 'certificates') {
+      return this.props.studioDetails.links.certificates.trim() !== '';
+    }
```

Comment:
Could you also add a condition for weekly highlights, since we do not want weekly highlights to show in the course checklist either. You will probably have to make changes to the course_quality API in edx-platform in order to pass through some kind of flag in the JSON response that indicates whether weekly highlights are enabled.

Reply:
@MichaelRoytman Next I will focus on weekly highlights after certificate issue is clear.